DebuggerGame/Assets/Scripts/BoardObject Scripts/CollidableObject.cs

```diff
@@ -4,5 +4,9 @@
 
 public class CollidableObject : BoardObject
 {
+    protected bool bugsCanPass = true;
```

Comment:
This should either be a virtual property (so that a subclass can change it, `virtual protected bool bugsCanPass => true;`) or serializable (so you can change it from the editor). I personally prefer the former but the latter is probably easier for now